environments/.env.development

```diff
@@ -1 +1 @@
-NEXT_PUBLIC_GITHUB_URL=https://api.github.com/
\ No newline at end of file
+NEXT_PUBLIC_ITUNES_URL=https://itunes.apple.com/
```

Comment:
Please add new a line at the end of files

Reply:
Noted and updated 👍🏻